File: webForm/MicrobeTraceNext/ClientApp/src/app/DTO/API-DTO/AddBlockResponse.ts

export class AddBlockResponse {
  tenantID!: string | undefined;
  userID!: string | undefined;
  ledgerName!: string | undefined;
  blockName!: string | undefined;
  blockProofHash!: string | undefined;
  statusMessage!: string | undefined;
  message: string;
  success: boolean;

  constructor() {}
}

File: webForm/BPaaSDTO/AddBlockResponse.cs
using System;
namespace BPaaSDTO
{
    public class AddBlockResponse
    {
        public string tenantID { get; set; }
        public string userID { get; set; }
        public string ledgerName { get; set; }
        public string blockName { get; set; }
        public string blockProofHash { get; set; }
        public string statusMessage { get; set; }
        public bool success { get; set; }
        public string message {get;set;}
    }
}
